ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -81,6 +82,7 @@ function TransactionListItemInner({
   const { supportsEIP1559 } = useGasFeeContext();
   const { openModal } = useTransactionModalContext();
   const testNetworkBackgroundColor = useSelector(getTestNetworkBackgroundColor);
+  const isSmartTransaction = useSelector(getIsSmartTransaction);
```

Comment:
Does the name of this suggest it's talking about the current transaction being rendered?

In which case, is this more explicit as something like `isSmartTransactionsActive`?

Reply:
This one actually checks if Smart Transactions are enabled in Settings by a user and a few other eligibility checks for Smart Transactions (Ethereum Mainnet, /health check, feature flag, etc.). If we want to update naming, I would prefer to do it in another PR and do it everywhere, since this selector is used a lot.